benchmarks/benchmark_throughput.py

```diff
@@ -229,7 +245,10 @@ def benchmark_throughput(
         )
     elif args.preset == "minimalistic":
         benchmark_throughput(
-            num_experts=1, num_clients=1, num_handlers=1, num_batches_per_client=args.num_batches_per_client
+            num_experts=1,
```

Comment:
nit: I'd suggest to revert such meaningless changes to minimize the diff

Reply:
This is the result of `black` work :)
Should we still left it as one line?

Reply:
Yeah, this line shouldn't be changed if you use `black` of the same version as specified in `requirements.txt` :)

Reply:
```
>>> black --version
<<< black, 22.3.0 (compiled: yes)
```

The only command for formatting I do is `black ./`.
Tried it just right now - `black` changes it to the current variant

Reply:
What if you run it from the root directory of the repo? My guess is that it fails to read the max line length setting from [pyproject.toml](https://github.com/learning-at-home/hivemind/blob/master/pyproject.toml#L7).

Reply:
It just got fixed now. Don't know what was the case